src/main/java/de/chojo/repbot/commands/Scan.java

```diff
@@ -41,11 +42,12 @@
     public static final int INTERVAL_MS = 2000;
     private static final int SCAN_THREADS = 10;
     private static final Logger log = getLogger(Scan.class);
+    private final ThreadGroup scanner = new ThreadGroup("Scanner");
     private final ScheduledExecutorService executorService = Executors.newScheduledThreadPool(SCAN_THREADS + 1,
             runnable -> {
-                var t = new Thread(runnable, "Scanner");
-                t.setUncaughtExceptionHandler((thread, err) -> log.error("Unhandled exception in Scanner Thread {}.", thread.getId(), err));
-                return t;
+                var thr = new Thread(scanner, runnable);
```

Comment:
Why is this variable name abbreviated?

Reply:
thread is already used in the lambda expression. That's the only reason.
I switch them, since the outer scope is more important.